apps/src/lib/levelbuilder/lesson-editor/activitiesEditorRedux.js

```diff
@@ -37,13 +37,15 @@ export const init = (
   activities,
   searchOptions,
   programmingEnvironments,
-  lessonExtrasAvailableForUnit
+  lessonExtrasAvailableForUnit,
+  isProfessionalLearningCourse
```

Comment:
then you can use destructuring syntax here:
```suggestion
  { 
    lessonExtrasAvailableForUnit,
    isProfessionalLearningCourse
  }
```
disclaimer: i haven't tried this out, and all the examples i looked at show ALL parameters being passed inside a single object. so, it's possible you'll have to move all the params inside a single object to get this to work.